Log a view suggestion for duplicate instrument conflicts (#4349)
open-telemetry/opentelemetry-go: CHANGELOG.md

- Fix `resource.WithHostID()` to not set an empty `host.id`. (#4317)
- Use the instrument identifying fields to cache aggregators and determine duplicate instrument registrations in `go.opentelemetry.io/otel/sdk/metric`. (#4337)
- Detect duplicate instruments for case-insensitive names in `go.opentelemetry.io/otel/sdk/metric`. (#4338)

> - Suggest a view to fix fixable instrument conflicts in `go.opentelemetry.io/otel/sdk/metric`. (#4349)

Reviewer: ```suggestion
- Suggest a view that fixes instrument conflicts in `go.opentelemetry.io/otel/sdk/metric`. (#4349)
```
